modules/seqexec/server/src/main/scala/seqexec/server/gpi/GPIController.scala

```diff
@@ -48,43 +156,33 @@ final case class GPIController(gpiClient: GPIClient[IO]) {
 
 object GPIController {
 
-  type Adc = edu.gemini.spModel.gemini.gpi.Gpi.Adc
-
-  implicit val adcEq: Eq[Adc] = Eq.by(_.displayValue)
-
-  type ObservingMode = edu.gemini.spModel.gemini.gpi.Gpi.ObservingMode
-
-  implicit val omEq: Eq[ObservingMode] = Eq.by(_.displayValue)
-
-  type Disperser = edu.gemini.spModel.gemini.gpi.Gpi.Disperser
-
-  implicit val dispEq: Eq[Disperser] = Eq.by(_.displayValue)
-
-  type Shutter = edu.gemini.spModel.gemini.gpi.Gpi.Shutter
+  implicit val adcEq: Eq[LegacyAdc] = Eq.by(_.displayValue)
 
-  implicit val shEq: Eq[Shutter] = Eq.by(_.displayValue)
+  implicit val omEq: Eq[LegacyObservingMode] = Eq.by(_.displayValue)
 
-  type ArtificialSource = edu.gemini.spModel.gemini.gpi.Gpi.ArtificialSource
+  implicit val dispEq: Eq[LegacyDisperser] = Eq.by(_.displayValue)
 
-  implicit val asEq: Eq[ArtificialSource] = Eq.by(_.displayValue)
+  implicit val shEq: Eq[LegacyShutter] = Eq.by(_.displayValue)
 
-  type PupilCamera = edu.gemini.spModel.gemini.gpi.Gpi.PupilCamera
+  implicit val asEq: Eq[LegacyArtificialSource] = Eq.by(_.displayValue)
 
-  implicit val pcEq: Eq[PupilCamera] = Eq.by(_.displayValue)
+  implicit val pcEq: Eq[LegacyPupilCamera] = Eq.by(_.displayValue)
 
   final case class AOFlags(useAo: Boolean,
                            useCal: Boolean,
                            aoOptimize: Boolean,
-                           alignFpm: Boolean)
+                           alignFpm: Boolean,
+                           magH: Double,
+                           magI: Double)
```

Comment:
Shouldn't we make more specific types for this information?

Reply:
i agree but I want to wait to have the `maginutde` from gem integrated